Assets/_BForBoss/_Utility/ObjectPoolerSample/SpawnParticles.cs

```diff
@@ -0,0 +1,67 @@
+using Perigon.Utility;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace BForBoss
+{
+    public class SpawnParticles : MonoBehaviour
```

Comment:
Is this like a demo script on how to use the `ObjectPooler`?

Reply:
Yeah, I left it in at the moment but I'll remove it before closing this

Reply:
The whole "ObjectPoolerSample" folder can be removed